session_csrf/tests.py

```diff
@@ -352,33 +351,60 @@ def test_surprising_characters(self):
             self.assertEqual(warner.call_count, 0)
 
 
-class ClientHandler(django.test.client.ClientHandler):
-    """
-    Handler that stores the real request object on the response.
-
-    Almost all the code comes from the parent class.
-    """
-
-    def __call__(self, environ):
-        # Set up middleware if needed. We couldn't do this earlier, because
-        # settings weren't available.
-        if self._request_middleware is None:
-            self.load_middleware()
-
-        signals.request_started.send(sender=self.__class__)
+def get_context_processors():
+    """Get context processors in a way that works for Django 1.4, 1.7, and 1.8"""
```

Comment:
Can you insert comments inside the try:except so as to remind us which one is for django 1.4 and which one is for `>1.4`.